docs/BR.md

```diff
@@ -1008,6 +1063,62 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
 
+#### 3.2.2.9 Multi-Perspective Issuance Corroboration
+
+Multi-Perspective Issuance Corroboration attempts to corroborate the determinations made by the Primary Network Perspective from multiple additional Network Perspectives before subscriber certificate issuance. This process can improve protection against equally-specific prefix Border Gateway Protocol (BGP) attacks or hijacks.
+
+Each corroborating Network Perspective MUST independently verify: 
+- the presence of the expected 1) Random Value, 2) Request Token, 3) IP Address, or 4) Contact Address, as required by the relied upon validation method specified in Sections 3.2.2.4 and 3.2.2.5, and
+- the CA's authority to issue to the requested domain(s) or IP address(es), as specified in Section 3.2.2.8
+
+[Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control) and [Section 3.2.2.5](#3225-authentication-for-an-ip-address) describe the validation methods that require the use of Multi-Perspective Issuance Corroboration and how a Network Perspective can corroborate the outcomes determined by the Primary Network Perspective. A CA MAY use corroborating evidence for CAA record compliance for a period equal to the period of validity for Validation of Domain Authorization or Control specified in [Section 4.2.1](#421-performing-identification-and-authentication-functions).
+
+Results or information obtained from one Network Perspective MUST NOT be reused or cached when performing validation through subsequent Network Perspectives (e.g., different Network Perspectives cannot rely on a shared DNS cache to prevent an adversary with control of traffic from one Network Perspective from poisoning the DNS cache used by other Network Perspectives). The network infrastructure providing Internet connectivity to a Network Perspective MAY be administered by the same organization providing the computational services required to operate the Network Perspective.
+
+CAs MAY immediately retry Multi-Perspective Issuance Corroboration using the same validation method or an alternative method (e.g., a CA can immediately retry validation using "Email to DNS TXT Contact" if "Agreed-Upon Change to Website - ACME" does not corroborate the outcome of Multi-Perspective Issuance Corroboration). There is no stipulation regarding the maximum number of validation attempts that may be performed in any period of time.
+
+The "Quorum Requirements" Table describes quorum requirements related to Multi-Perspective Issuance Corroboration. Network Perspectives are considered distinct when the straight-line distance between the two States, Provinces, or Countries they reside in is at least 500 km.
+
+Table: Quorum Requirements
+
+| # of Distinct Network Perspectives Used | # of Allowed non-Corroborations |
+| --- | --- |
+| 2-5 |  1  |
```

Comment:
In general I agree with you that two perspectives with one non-corroborating is a relatively week security improvement. There is still some improvement as the attacker now as to fool the CA's primary infrastructure plus an additional network perspective, but this is not particularly hard.

I think the original motivation for this particular policy was simply to promote an easy transition. This configuration requires a CA to deploy as few remote perspectives as possible and permits issuance under quite liberal policies which can help ensure MPDV does not interfere with benign operations. Per the timeline text, this policy can only be used between June and December 2025 at which point two remote perspectives is no longer permitted.

Personally I am fine leaving or changing this line. Since this configuration is only active for 6 months, I do not think there is a major security incentive one way or the other.

Reply:
To me, what change is needed in this section comes down to what counts towards the total "# of Distinct Network Perspectives Used".  I had assumed the total of distinct network perspectives includes all network perspectives used _including_ the Primary Network Perspective.  Your comment sounds like you are only counting the additional perspectives towards the total.

If your interpretation is correct then I think the numbers in the table are fine.  In this situation, your assessment of the security benefit is completely fair as you're right to say that an attacker would need to fool the primary infrastructure and an additional network perspective.  In this case, I think some language needs changing in the above text to make this much clearer because I didn't understand this to be what was meant by the "# of Distinct Network Perspectives Used".

If the total in the left hand column does include the Primary Network Perspective, as I assumed it does, then I feel like my comment stands and where there is only the Primary Network Perspective and a single other Network Perspective then the two perspectives have to agree to allow issuance.

Reply:
I am glad this can be clarified. I was interpreting the table as "# of Distinct Network Perspectives Used in Addition to the Primary Perspective". Maybe a clean resolution would be to clarify this in the column description for the table:

```
| # of Distinct Network Perspectives Used in Addition to the Primary Network Perspective | # of Allowed non-Corroborations |
```

@robplee let me know if this adds sufficient clarity to the table.

Reply:
So, that's one option that would definitely add clarity.  I wonder if a cleaner solution would be to come up with a name for the non-Primary Network Perspectives then this column could be the # of Distinct <Word> Network Perspectives and it'd be clearer that it's counting the non-primary network perspectives.  Would "Remote" or "Secondary" work?

Reply:
Sorry for the slow response to these comments.

@robplee - I agree that we could do a better job of making it clear that the perspectives used for MPIC are separate from the primary. 

What do you think about [these](https://github.com/ryancdickson/staging/commit/46f6059bcee06384e4600c66aa05745a9fe7a3d4) changes?

Reply:
@ryancdickson change looks great, consider this thread resolved :)

Reply:
Appreciate your review and feedback, @robplee!